### DISCUSSION TITLE
Proposal: Add a "continuous" attribute to the <podcast:liveItem> tag

### DISCUSSION THREAD

**Discussion by mitchdowney** [Podcastindex-org/podcast-namespace#465] [Enhancement Proposal]
Currently the <podcast:liveItem> spec says the `start` attribute is required, but the start date/time of a 24/7 continuously running livestream isn't very useful for rendering in client apps. 

For example, "100% Retro Live Stream 24/7" is a continuously running radio station that uses the <podcast:liveItem> tag.

<img width="685" alt="image" src="https://user-images.githubusercontent.com/2262919/198544403-8ce34b20-b11c-44a5-a6e6-d274b457e17f.png">

If there was an attribute that told us this is not a "one-off" livestream, but a continuously running livestream, then client apps could choose not to render the `start` time. How about a "continuous" attribute in the liveItem tag?

I don't have a strong preference for a name. I just think some sort of indicator for 24/7 livestreams would be useful.

**Comment by theDanielJLewis** [comment]
This might not be used much, but it still seems like a good idea. 

Maybe simply the omission of `start` would trigger apps to interpret the stream as currently live, and with no end date, it would simply be always live until LIT is removed.

**Comment by daveajones** [comment]
> Maybe simply the omission of `start` would trigger apps to interpret the stream as currently live, and with no end date, it would simply be always live until LIT is removed.

Interesting idea.  Omitting start and end times could indicate continuous.  I like that.
